app/Services/TransactionFlowService.php

```
@@ -45,4 +47,44 @@ public static function updateTransactionFlow($paymentRequestId, $transactionFlow
+            ],
+            PaymentRequestType::PAYROLL->value => [
+                PaymentRequestType::PRF->value,
+                PaymentRequestType::PO->value
+            ],
+            PaymentRequestType::PO->value => [
+                PaymentRequestType::PRF->value,
+                PaymentRequestType::PAYROLL->value
+            ],
+            default => []
+        };
+        $transactionFlowTemplates = TransactionFlowModel::whereNotIn('category', $excludedCategories)
```

PHPMD (2.15.0)

Avoid excessively long variable names like $transactionFlowTemplates. Keep variable name length under 20. (Naming Rules)

(LongVariable)